### PR TITLE
Update _new.md

### PR DESCRIPTION
**Pull request by yanshiyason**
typo

### CODE REVIEW

**Comment by yanshiyason** [comment]
https://gobuffalo.io/en/docs/apis

This part of the documentation has a typo: https://gobuffalo.io/en/docs/apis#buffalo-new-coke-1

The generated session name is `_pepsi_session` which means the app name should be `pepsi` but it is still `coke`

**Comment by yanshiyason** [comment]
@markbates sorry, I am new to buffalo.. is the documentation correct?

**Comment by markbates** [comment]
Yes. :)

-----------
Mark Bates

**Comment by markbates** [comment]
Oh, I just realized that you had a serious change in there, as well as a joke change.

File: templates/docs/apis/_new.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ API applications have `actions/app.go` and `actions/render.go` files that are a
 <%= partial("docs/apis/api_app.md") %>
 <%= partial("docs/apis/api_render.md") %>
 
-<h5><code>buffalo new coke</code></h5>
+<h5><code>buffalo new pepsi</code></h5>
 
 <%= partial("docs/apis/web_app.md") %> 
-<%= partial("docs/apis/web_render.md") %>
\ No newline at end of file
+<%= partial("docs/apis/web_render.md") %>
```
Comment (yanshiyason) [inline templates/docs/apis/_new.md]: actually, this line shouldn't be updated.. sorry, I used the github web UI to make the change... let me remove this line

**Comment by yanshiyason** [comment]
image

image

@markbates 

Sorry, I thought this was a mistake. The name of the session is `_pepsi_session` even though the app name is `coke`

```
$ buffalo new foobar
cat ./foobar/actions/app.go | grep SessionName
			SessionName: "_foobar_session",
```

**Comment by yanshiyason** [comment]
@lukasschlueter Sounds good! I changed the session name to coke.

**Pull request merged**